README.md

```diff
@@ -9,36 +9,42 @@ NCSA Common Puppet Profiles - configure GitLab service
 
 1. [Description](#description)
 1. [Setup - The basics of getting started with profile_gitlab](#setup)
-    * [What profile_gitlab affects](#what-profile_gitlab-affects)
-    * [Setup requirements](#setup-requirements)
-    * [Beginning with profile_gitlab](#beginning-with-profile_gitlab)
 1. [Usage - Configuration options and additional functionality](#usage)
-1. [Limitations - OS compatibility, etc.](#limitations)
-1. [Development - Guide for contributing to the module](#development)
+1. [Dependencies](#dependencies)
+1. [Reference](#reference)
+
 
 ## Description
 
 This puppet profile customizes a host to install and configure GitLab service
 
+
 ## Setup
 
 Include profile_gitlab in a puppet profile file:
 ```
 include ::profile_gitlab
 ```
 
+
 ## Usage
 
 The goal is that no paramters are required to be set. The default paramters should work for most NCSA deployments out of the box.
 
-## Reference
 
-See: [REFERENCE.md](REFERENCE.md)
+## Dependencies
```

Comment:
Does ::gitlab need to be included as a dependency? Unless I missed something it doesn't look like that class is defined in this module.

Reply:
Yes. Thanks.